src/Components/Layout/index.js

```diff
@@ -23,7 +23,9 @@ function Layout() {
           <Route exact path="/" component={Home} />
           <Route path="/activities" component={Activities} />
           <Route path="/admins" component={Admins} />
-          <Route path="/classes" component={Classes} />
+          <Route exact path="/classes" component={Classes} />
+          <Route exact path="/classes/create" component={ClassForm} />
```

Comment:
```suggestion
          <Route exact path="/classes/add" component={ClassForm} />
```
To maintain consistency with the team, the route should be 'classes/add'